### PR TITLE
Fix Natspec

### PR DESCRIPTION
**Pull request by Vectorized**
- Fix comment style for some Natspec. 
- Renamed `minter` to `to` for the `totalPrice` functions for clarity.

### CODE REVIEW

**Comment by gigamesh** [comment]
@Vectorized I like your suggestion of making a 1.1 branch - should this go into that?

**Review by gigamesh — APPROVED** [review]

File: contracts/core/interfaces/IMinterModule.sol [hunk 1]
```diff
@@ -87,7 +87,7 @@ interface IMinterModule is IERC165 {
     event Minted(
         address indexed edition,
         uint128 indexed mintId,
-        address indexed buyer,
+        address indexed to,
```
Comment (vigneshka) [inline contracts/core/interfaces/IMinterModule.sol:90]: is this a breaking change for indexers?

**Review by vigneshka — CHANGES_REQUESTED** [review]
small q on changing existing events

File: contracts/core/interfaces/IMinterModule.sol (hunk 1 above)
Comment (Vectorized) [reply contracts/core/interfaces/IMinterModule.sol:90]: Nope. The signature only depends on the type of the args.

See: https://github.com/chiru-labs/ERC721A/blob/1843596cf863557fcd3bf0105222a7c29690af5c/contracts/ERC721A.sol#L89

**Review by vigneshka — APPROVED** [review]

**Pull request merged**